parm/config/config.base.emc.dyn

```diff
@@ -11,6 +11,7 @@ export machine="@MACHINE@"
 # EMC parallel or NCO production
 export RUN_ENVIR="emc"
 
+
```

Comment:
Remove extra line added, thanks!

Reply:
Will do. To clarify, the `DO_EFSOI` switch is to determine if the entire experiment is doing EFSOI. A separate `EFSOI_TASK` is used to flag tasks that share code with regular tasks. This is used elsewhere, but can be used to advantage as you suggest.